tests/conftest.py

```diff
@@ -114,7 +114,13 @@ def ssh_client_session(sshd_addr, ssh_clientkey_path):
         host_key_checking=False,
         look_for_keys=False,
     )
-    return ssh_session
+    try:  # noqa: WPS501
+        yield ssh_session
+    finally:
+        if hasattr(ssh_session, 'sftp') and ssh_session.sftp is not None:
```

Comment:
Why would this attr be missing? If there are cases where it's true, it's a design flaw.